src/splunk-events.ts

```diff
@@ -298,62 +362,102 @@ export default class SplunkEvents {
     })
       .then(() => {
         if (this.debug) {
-          console.log(
-            `${this.pendingEvents.length} events successfuly sent to splunk`
-          )
+          console.log(`${events.length} events successfuly sent to splunk`)
         }
+      })
+      .catch((e) => {
+        if (this.debug) {
+          console.warn('Error sending events to splunk.', e)
+        }
+
+        throw e
+      })
+  }
+
+  private _backoffFlush = () => {
```

Comment:
why the underscore prefix?

Reply:
I just wanted to differentiate it from the "public" interface, but I can change this if you want :-)

Reply:
for some reason, it looks natural to me. If you think this isn't going to be weird for other JS devs, no problem